src/pvrclient-nextpvr.cpp

```diff
@@ -34,10 +34,10 @@
 #include "tinyxml.h"
 
 #if defined(TARGET_WINDOWS)
-  #define atoll(S) _atoi64(S) 
```

Comment:
Not a fan of gratuitous white space changes mixed with code changes.

Reply:
I am not a fan of white space being there in the first place but I will revert these